lab_1_keywords_tfidf/start.py

```diff
@@ -31,6 +33,40 @@
     with open(CORPUS_FREQ_PATH, 'r', encoding='utf-8') as file:
         corpus_freqs = json.load(file)
 
-    RESULT = None
-    # DO NOT REMOVE NEXT LINE - KEEP IT INTENTIONALLY LAST
-    assert RESULT, 'Keywords are not extracted'
+        if target_text:
+            tokens = clean_and_tokenize(target_text)
+
+        if tokens and stop_words:
+            tokens_without_stop_words = remove_stop_words(tokens, stop_words)
+
+        if tokens_without_stop_words:
+            frequencies = calculate_frequencies(tokens_without_stop_words)
+
+        if frequencies:
+            get_top_n(frequencies, 10)
```

Comment:
according to the task description you only need to show this function work of tfidf and chi dictionaries
this way you'll be able to fix your typings issues in CI